docs/building-a-form/common-definitions.md

```diff
@@ -1,40 +1,29 @@
-
-### Common widgets
-
-These common widgets are included in the us-forms-system by default. Set these widgets in a config file while building your form. Some are associated with particular schema types or formats. There are additional widgets available, but the us-forms-system uses definitions they're included in, rather than just the field.
-
-To use a widget, set `ui:widget` for a field to the name of the widget. Widgets are located in [/src/js/widgets](../../src/js/widgets).
+- Simple definitions are provided as `schema` and `uiSchema` objects that you can import and overwrite to customize.
+- More complex definitions are functions that require certain parameters.
 
-Widget       | How it's used
------------- | -------------
-`ArrayCountWidget` | In the `uiSchema`, specify `'ui:widget': ArrayCountWidget` for the given field
-`CheckboxWidget` | Default widget for a schema that specifies `type: 'boolean'`
-`CurrencyWidget` | In the `uiSchema`, specify `'ui:widget': CurrencyWidget` for the given field
-`DateWidget` | In the `uiSchema`, specify `'ui:widget': 'date'` for the given field
-`EmailWidget` | In the `uiSchema`, specify `'ui:widget': 'email'` for the given field
-`PhoneNumberWidget` | In the `uiSchema`, specify `'ui:widget': PhoneNumberWidget` for the given field
-`RadioWidget` | In the `uiSchema`, specify `'ui:widget': 'radio'` for the given field
-`SelectWidget` | Default widget for a schema that specifies `type: 'string'` with an `enum` property
-`SSNWidget` | In the `uiSchema`, specify `'ui:widget': SSNWidget` for the given field
-`TextWidget` | Default widget for a schema that specifies `type: 'string'`
-`YesNoWidget` | In the `uiSchema`, specify `'ui:widget': 'yesNo'` for the given field
+### Using definitions
 
-** Note: Some widgets are passed as a string (e.g., `'ui: widget': 'date'`), while others are passed as the component itself (e.g., `'ui:widget': CurrencyWidget`). Widgets from the [react-jsonschema-form library](./about-the-us-forms-system-library#understanding-react-jsonschema-form-rjsf) includes string mappings. Those created specifically for us-forms-system do not have these mappings, and therefore must be specified by passing the component for the widget directly to the config.
+To use a definition, import it near the top of the file you want to reference it in:
 
-### Common definitions
+```js
+import currencyUI from 'us-forms-system/lib/js/definitions/currency';
+```
 
-These definitions address form field patterns that are more complex than widgets and include common label text, validation, or field components. Definitions are located in [/src/js/definitions](../src/js/definitions).
+Then, call it to add all the `uiSchema` definitions:
 
-- Simple definitions are provided as `schema` and `uiSchema` objects that you can import and overwrite to customize.
-- More complex definitions are functions that require certain parameters.
+```js
+uiSchema: {
+  ...
+  monthlyWages: currencyUI('Monthly wages')
```

Comment:
For this specific example, I might add text below it to clarify what this is:

"In this example, the definition is a function that takes the title for that field, which is used to populate the `'ui:title'` property in `uiSchema`."

Reply:
That makes sense. I was mentioning to @bernars-usa that this is just by convention and it would be possible for others to define these items with more than one arg or a multi-property object as an argument. Even the definitions we create take different numbers and orders of arguments so you have to look them up in the code at the moment.